lib/assets/javascripts/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "turbograft",
+  "version": "1.0.0",
+  "description": "Turbograft extends [Turbolinks](https://github.com/rails/turbolinks), allowing you to perform partial page refreshes.",
+  "scripts": {
+    "test": "dev test"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Shopify/turbograft.git"
+  },
+  "author": "Shopify",
+  "license": "MIT",
+  "bugs": {
+    "url": "https://github.com/Shopify/turbograft/issues"
+  },
+  "homepage": "https://github.com/Shopify/turbograft#readme",
+  "dependencies": {
+    "promise-polyfill": "6.0.1"
```

Comment:
This approach doesn't work, by the way, unless you are checking your node modules in to the repo. I think we're overthinking this — just add to the README that you need a Promise polyfill, and act as if you have the polyfill available (and actually polyfill it for tests).